rubrics/api/src/main/java/org/sakaiproject/rubrics/api/repository/EvaluationRepository.java

```diff
@@ -37,4 +37,6 @@ public interface EvaluationRepository extends SpringCrudRepository<Evaluation, L
     int deleteByAssociationIdAndEvaluatedItemId(Long associationId, String evaluatedItemId);
     int deleteByToolItemRubricAssociation_Id(Long associationId);
     int deleteByOwnerId(String ownerId);
+    List<Evaluation> findByOwnerId(String ownerId);
+    void deleteByList(List<Evaluation> evaluations);
```

Comment:
this method probably isn't needed as it's not actually deleting by a list but rather it just performs a foreach and calls delete individually, which is a simple enough lambda already.
If I saw this on method on a repository I would expect it to perform a bulk delete.

Reply:
Yes, makes sense. I updated the PR